2024Robot_Team3484/src/main/cpp/subsystems/ClawSubsystem.cpp

```diff
@@ -0,0 +1,49 @@
+#include "subsystems/ClawSubsystem.h"
+
+ClawSubsystem::ClawSubsystem(
+    int left_motor_can_id,
+    int right_motor_can_id,
+    int left_sensor_di_ch,
+    int right_sensor_di_ch
+
+    ) : 
+        _left_climb_motor{left_motor_can_id},
+        _right_climb_motor{right_motor_can_id},
+        _left_motor_sensor{left_sensor_di_ch},
+        _right_motor_sensor{right_sensor_di_ch}
+    {
+
+    _right_climb_motor.ConfigFactoryDefault();
+    _left_climb_motor.ConfigFactoryDefault();
+
+    _left_climb_motor.SetNeutralMode(motorcontrol::Brake);
+    _right_climb_motor.SetNeutralMode(motorcontrol::Brake);
+
+    _left_climb_motor.SetInverted(ClawConstants::MOTOR_INVERTED);
+    _right_climb_motor.SetInverted(ClawConstants::MOTOR_INVERTED);
```

Comment:
Todo: Verify motor invertedness. Nothing to do here, just a note for later. It's possible one side is inverted and the other is not.